aptos-node/src/lib.rs

```diff
@@ -181,6 +182,70 @@ impl AptosNodeArgs {
     }
 }
 
+fn load_remote_config(initial_config: &NodeConfig) -> anyhow::Result<Option<NodeConfig>> {
+    let data_dir = initial_config.get_data_dir();
+    let config_dir = data_dir.join("remote_configs");
+    if !config_dir.is_dir() {
+        warn!("{:?} is not a dir. using initial config", config_dir);
+        return Ok(None);
+    }
+    let mut entries = fs::read_dir(config_dir.clone())?
+        .filter_map(|res| {
+            let Ok(entry) = res else {
+                return None;
+            };
+            let file_name = entry.file_name().into_string().ok()?;
+            if file_name.starts_with("remote_config_v") {
+                return Some(file_name);
+            }
+            None
+        })
+        .collect::<Vec<_>>();
+    entries.sort();
```

Comment:
Doesn't seem necessary since we go through all entries.

If we were to rely on this, since this is an alphabetical sort, we'll have to pad 0's. e.g., start with v000 to support 1000 versions